docs/papers/architecture-whitepaper.md

```diff
@@ -0,0 +1,1739 @@
+
+```mermaid
+sequenceDiagram
+  participant w1 as Wallet 1
+  participant a as Wallet Addressing
+  participant w2 as Wallet 2
+  
+  w1->>+a: Register(w1_name, w1_protocol)
+  a-->>-w1: w1_address
+  w2->>+a: Register(w2_name, w1_protocol)
+  a-->>-w2: w2_address
+  
+  w1->>+a: Lookup(w2_name)
+  a-->>-w1: w2_address, w2_protocol
+  
+  w1->>+w2: Connect<w2_address, w2_protocol>(w1_name)
+  w2->>+a: Lookup(w1_name)
+  a-->>-w2: w1_address, w1_protocol
+  w2->>-w1: ACK<w1_address, w1_protocol>(w2_name)
+```
+
+###### Sub-capabilities
+- **Wallet Addressing** - Ability to uniquely identify a wallet to ensure that communications are routed correctly.
+- **Peer Discovery** - Enables wallets to identify and establish connections with other wallets.
+- **Secure Communication Protocols** - Ensure the confidentiality, integrity, and authenticity of P2P communications.
+
+###### Considerations
+- **Autonomy and Control** - Users maintain full control and ownership of their data and transactions, with the ability to transact directly with peers without third-party intervention.
+- **In-Person Processing** - P2P communication may be used in cases where data needs to be presented in person. Consider how contactless protocols may be utilized in these cases. In addition, consider how the wallet can be used without unlocking the underlying device or other items in the wallet.
+
+
+## Technology Enablers
+A **Technology Enabler** facilitates the implementation of the overall solution. Technology enablers can include software libraries, development frameworks, hardware components, cloud services, protocols, and standards. In this section, we break the enablers into cross-cutting enablers (those that support multiple solutions), identity enablers (those that support identity solutions), object enablers (those that support digital object solutions), and money enablers (those that support money solutions).
+
+### Cross Cutting Enablers
+- HTTPS/TLS
+- Public Key Cryptography
+- [Decentralized Key Recovery Protocols](https://derecalliance.org/)
+- Zero-Knowledge Proofs
+- Near Field Communication
+- Bluetooth Low-Energy
+- QR Codes
+- Magnetic Secure Transmission
+- Key Vaults
+- Secure Enclaves (TEE)
+- Hardware Security Modules
+- Biometrics
+- OAuth/OpenID Connect
+- FIDO2/Client-to-Authenticator Protocol (CTAP)
+- Trusted Service Managers (TSM)
+
+### Identity Enablers
+- DIDs
+- OIDC SIOP
+- Credential Formats
+    - W3C Verifiable Credentials
+    - AnonCreds
+    - ISO mDL
+    - ICAO DTC
+    - SD-JWT
+- P2P Protocols
+    - DIDComm
+    - WebRTC
+- Data Exchange Protocols
+    - OID4VC
+    - VC-API
+    - CHAPI
+    - Aries Interop Profile
+- Verifiable Data Registries
+
+### Objects Enablers
+- Token Taxonomy Framework
+- Ethereum Token Standards
```

Comment:
Is this section open to other standards or to be made agnostic to specific protocols? There are a couple blockchain interoperability working groups that might be able to link specification that takes into consideration Virtual Machine based standards or native ledger (no virtual machine) standards. I'll try to find something we could propose here

Reply:
This section is open to other standards. If you can find a way to make it agnostic, even better. Looking forward to your proposal.